components/scream/data/scream_input.yaml

```diff
@@ -61,31 +77,32 @@ HOMME:
     partmethod        : 4
     cubed_sphere_map  : 0
     topology          : cube
-    test_case         : jw_baroclinic
     u_perturb         : 1
     rotate_grid       : 0
     ne                : 4
     statefreq         : 9999
     mesh_file         : /dev/null
     tstep             : 300
-    nu                : 7.0e+15
-    nu_div            : 1.0e+15
-    nu_p              : 7.0e+15
-    nu_q              : 7.0e+15
-    nu_s              : -1
-    nu_top            : 2.5e+5
+    nu                :  .000000034
+    nu_div            : -1 
+    nu_p              : .000000034
+    nu_q              : .000000034
+    nu_s              : .000000034
+    nu_top            : 
     se_ftype          : 0
     limiter_option    : 9
-    vert_remap_q_alg  : 1
-    hypervis_scaling  : 0
+    vert_remap_q_alg  : 10
+    hypervis_scaling  : 3.0
     hypervis_order    : 2
-    hypervis_subcycle : 2
-    hypervis_subcycle_tom : 0
-    theta_hydrostatic_mode : false
-    theta_advect_form : 0
-    tstep_type        : 10
-    dt_remap_factor   : 3
+    hypervis_subcycle : 1
+    hypervis_subcycle_tom : 1
+    theta_hydrostatic_mode : true
+    theta_advect_form : 1
+    tstep_type        : 5
+    dt_remap_factor   : 1
     dt_tracer_factor  : 1
+    transport_alg     : 12
```

Comment:
Was this working for you? It looks to me like `transport_alg` option hasn't been added to our HOMME namelist read routine.

Reply:
I think you are right.  This option may not be used.  To be safe I copied everything I had in the new namelist.  But I can remove this.

Reply:
I guess if it passes that's fine, but including this one for me was throwing zeros into the rest of the fields.